app/src/main/java/nsit/app/com/nsitapp/AboutUs2.java

```diff
@@ -0,0 +1,98 @@
+package nsit.app.com.nsitapp;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.LoaderManager;
+import android.support.v4.content.Loader;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import adapters.MemberAdapter;
+import functions.ButtonAnimation;
+
+/**
+ * Created by Kartik Kwatra on 26-08-2017.
+ */
+
+public class AboutUs2 extends Fragment implements LoaderManager.LoaderCallbacks<List<AboutUsMember>> {
+
+
+    private static MemberAdapter mAdapter;
+    private static final String LOG_TAG = AboutUs2.class.getName();
+    private static final int LOADER_ID = 1;
+    private Activity aboutUsAct;
+    private View rootView;
+
+    public AboutUs2() {
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
+        rootView = inflater.inflate(R.layout.aboutus_activity, container, false);
+        ListView aboutUsListView = (ListView) rootView.findViewById(R.id.teamList);
+        aboutUsAct = getActivity();
+        View footerView = ((LayoutInflater) aboutUsAct.getApplicationContext().getSystemService(Context.LAYOUT_INFLATER_SERVICE)).inflate(R.layout.aboutus_footer, null, false);
+        mAdapter = new MemberAdapter(getActivity(), new ArrayList<AboutUsMember>());
+        aboutUsListView.setAdapter(mAdapter);
+        aboutUsListView.addFooterView(footerView);
+        TextView rep = (TextView) rootView.findViewById(R.id.GoToRepo);
+        TextView con = (TextView) rootView.findViewById(R.id.cont);
```

Comment:
Make the variable name more informative. rep & con doesn't actually tell, what these are for.

Reply:
These variable names were picked up from the previous code. I'll change them.